### DISCUSSION TITLE
Approaches to iterating WCAG 3

### DISCUSSION THREAD

**Discussion by rachaelbradley** [w3c/wcag3#18] [General]
One of the goals for WCAG 3 is to create a more flexible maintenance model.  This discussion thread is to discuss ways to make updates more quickly to keep up with changing technology.

**Comment by rachaelbradley** [comment]
One option was using levels (Bronze, Silver and Gold) to provide a way to add new guidance.  Guidance could enter at the Gold or Silver level and then migrate to Bronze as the guidance has better support or is more quantifiable. How well this will work will depend on the conformance model. 

This was discussed at CSUN 2023.  [Matrix Model Details](https://docs.google.com/document/d/16ooY_O3YWHG11OF7yFmdVnHYbUDTp4Csij4-6kKg4ZQ/edit#heading=h.lgq31qj3f6o1)

**Comment by ljm22** [comment]
From thread in linkedin: https://www.linkedin.com/posts/lwatson_w3c-accessibility-guidelines-wcag-30-activity-7089262646587195393-Cvmz

I said: "I guess there is a tension between the needs of practitioners and developers (frequent small iteration is good) and legislators and lobbyists of said legislators (frequent small iteration is bad)", Eric Eggert responded: "I have heard that argument, but I have not found it to be reflected in truth. EN301549 – the European Standard – has changed a lot in relatively short time. And one approach would be to iterate quickly for practitioners and then bundle it up into major versions for legislation. I guess I will not get around writing a blog post about this :-D" to which I responded: "yes ok, minor and major versions makes sense, with legislation typically referencing… oooh… the n.1 version each time (on the basis that the n.0 version always needs tidying up)."

On furhter consideration then, I suggest that the iteration method needs to allow for legislation to reference a static version, and then update as new ones are iterated. 3.0, 3.1, 3.2 would allow for this, but should have a faster pace of iteration (3.0.1, 3.0.2 perhaps) than was the case between 2.0 and 2.1. 

Also ideally would want diffs between different versions -- this allows for testers who already believe their website or SaaS to be at 3.0.1 AA compliant, for example, to add in the additional requirements to hit 3.0.2.

(and hi, @shawna-slh, hope this finds you well!)

**Comment by yatil** [comment]
My thoughts are collected in the following works:

- [Blog post: Fix Web Accessibility Systematically](https://yatil.net/blog/fix-web-accessibility-systematically)
- [This talk](https://youtube.com/watch?v=PZYiweHuo_I), especially the [last 30 minutes](https://youtube.com/watch?v=PZYiweHuo_I&t=2673) (video)
- [Blog post: Do we need WCAG 3 now?](https://yatil.net/blog/do-we-need-wcag-3-now)
